ext/web/06_streams.js

```diff
@@ -1653,35 +1666,48 @@
     }
   }
 
+  function isTypedArray(view) {
+    return ObjectPrototypeIsPrototypeOf(Int8ArrayPrototype, view) ||
```

Comment:
If think the preferred way to assert TypedArrayness is from `ArrayBufferIsView` and `TypedArrayPrototypeGetSymbolToStringTag`.

Reply:
Just `TypedArrayPrototypeGetSymbolToStringTag(view) !== undefined` is fine.